src/installer/pkg/projects/netcoreapp/sfx/Microsoft.NETCore.App.SharedFx.sfxproj

```diff
@@ -2,6 +2,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
+    <NoWarn>$(NoWarn);NU1603</NoWarn>
```

Comment:
We should not suppress this.  That warning indicates a newer version of a package was restored rather than the one specified.  We should fix the version.

Reply:
i checked and we can remove this now.